core/src/main/java/tachyon/UnderFileSystem.java

```diff
@@ -62,6 +64,8 @@ public static UnderFileSystem get(String path) {
    * @return null for any unknown scheme.
```

Comment:
Not directly related to this change, but could you update the documentation here? (throws IllegalArgumentException for unknown schemes). Also, could you add the throws annotation to the two functions since checkArgument will throw IllegalArgumentException?